src/components/dashboard/sidebar/SubsidiesSummary.jsx

```diff
@@ -113,25 +188,25 @@ const SubsidiesSummary = ({
             className="border-0 shadow-none"
```

Comment:
[curious] These class names were removed from `SubscriptionSummaryCard` and `LearnerCreditSummaryCard`. Does it need to be removed from `CouponCodesSummaryCard` as well?